blog/electron-15-0.md

```diff
@@ -0,0 +1,94 @@
+The Electron team is excited to announce the release of Electron 15.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://electronjs.org/releases/stable). Continue reading for details about this release and please share any feedback you have!
+
+## Notable Changes
+
+### Electron Release Cadence Change
+
+Starting with Electron 15, Electron will release a new major stable version every 8 weeks. You can read the [full details here](https://www.electronjs.org/blog/8-week-cadence).
+
+Additionally, Electron will be changing supported versions from latest three versions to latest four versions until May 2022. [See our versioning document](https://electronjs.org/docs/tutorial/electron-versioning)for more detailed information about versioning in Electron.
+
+### Stack Changes
+
+* Chromium `94`
+    * [New in Chrome 94](https://developer.chrome.com/blog/new-in-chrome-94/)
+* Node.js `16.5.0`
+    * [Node 16.5.0 blog post](https://nodejs.org/en/blog/release/v16.5.0/)
+* V8 `9.4`
+    * [V8 9.4 blog post](https://v8.dev/blog/v8-release-94)
+
+### Highlight Features
+
+* `nativeWindowOpen: true` is no longer experimental, and is now the default.
+* Added 'frame-created' event to `WebContents` which emits when a frame is created in the page. [#30801](https://github.com/electron/electron/pull/30801) 
+* Added resize `edge` info to `BrowserWindow`'s `will-resize` event. [#29199](https://github.com/electron/electron/pull/29199)
+
+See the [15.0.0 release notes](https://github.com/electron/electron/releases/tag/v15.0.0) for a full list of new features and changes.
+
+## Breaking Changes
+
+Below are breaking changes introduced in Electron 15. More information about these and future changes can be found on the [Planned Breaking Changes](https://www.electronjs.org/docs/breaking-changes) page.
+
+### Default Changed: nativeWindowOpen defaults to true
+
+Prior to Electron 15, `window.open` was by default shimmed to use `BrowserWindowProxy`. This meant that `window.open('about:blank')` did not work to open synchronously scriptable child windows, among other incompatibilities. `nativeWindowOpen: true` is no longer experimental, and is now the default.
+
+See the documentation for [window.open](https://www.electronjs.org/docs/api/window-open) in Electron for more details.
+
+
+## API Changes
+
+* Added 'frame-created' event to `WebContents` which emits when a frame is created in the page. [#30801](https://github.com/electron/electron/pull/30801) 
+* Added `safeStorage` string encryption API. [#30430](https://github.com/electron/electron/pull/30430) 
+* Added `signal` option to `dialog.showMessageBox`. [#26102](https://github.com/electron/electron/pull/26102) 
+* Added an [Electron Fuse](https://www.electronjs.org/docs/tutorial/fuses) for enforcing code signatures on the `app.asar` file your application loads.  Requires the latest `asar` module. [#30900](https://github.com/electron/electron/pull/30900) 
```

Comment:
Might be clear to mention which semver of asar is required here.

Reply:
```suggestion
* Added an [Electron Fuse](https://www.electronjs.org/docs/latest/tutorial/fuses) for enforcing code signatures on the `app.asar` file your application loads.  Requires the latest `asar` module. [#30900](https://github.com/electron/electron/pull/30900) 
```

Reply:
@erickzhao Added asar semver version v3.1.0 or higher 👍